docs/public/_redirects

```diff
@@ -3,6 +3,7 @@
 /inbox* / 301
 /trash / 301
 /spam / 301
+/sign-up / 301
```

Comment:
To fix https://app.ahrefs.com/site-audit/3524616/33/data-explorer?columns=pageRating%2Curl%2Ctraffic%2ChttpCode%2Cdepth%2Ccompliant%2CincomingAllLinks%2Corigin&filterCollapsed=true&filterId=91013c8aafad4fe1f7a0ad504eccad42&issueId=c64da643-d0f4-11e7-8ed1-001e67ed4656&sorting=-pageRating. We might get a MUI real page in the future.